### ISSUE TITLE
Update to 4.4.9 requires PHP 7.1

### ISSUE THREAD

**Issue by calculon102** [contao/core-bundle#1269]
I just migrated my production environment from Contao 3.5 LTS to Contao 4.4.8 LTS (managed). It runs with PHP 7.0. When I did test an update to 4.4.9 I got
```
Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Return value of Doctrine\Common\Annotations\AnnotationRegistry::registerFile() must be an instance of Doctrine\Common\Annotations\void, none returned in ***/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationRegistry.php:67
```
when making web-requests thereafter.

Some research showed that doctrine/annotation starting from 1.5 requires PHP 7.1, which is again required by some other doctrine-bundle.

As far as I understand the 4.4 LTS-branch is wanted to stay PHP 7.0-compatible, so this is a bug, right? At least for me this is a showstopper at the moment.

P.S.
Not sure if this issue is for core-bundle or managed-edition. Sorry, if incorrect here.

**Comment by fritzmg** [comment]
How did you update to Contao 4.4.9?

**Comment by calculon102** [comment]
Did `composer update` on an existing install and tried separately `composer install --no-dev` on a new install. I did this on my local machine and copied the results on my production server after that.  My hosted production server has not enough memory to run composer itself. At least not for contao-installs.

But there was my fault. I run an Ubuntu 17.10 dev-machine which installs PHP 7.1 as default. So when I run composer locally it detects PHP 7.1 and installs dependencies with to this version as minimum.

I did just install PHP 7.0 and ran `composer install --no-dev`  with that. Now I have a PHP 7.0 compatible Contao 4.4.9 install.

So problem solved. This was not a contao-issue, but a misunderstanding of how composer works on my side.

For others running into this issue: I had to use [this PPA](https://launchpad.net/~ondrej/+archive/ubuntu/php-qa) to install PHP7.0 on Ubuntu 17.10 (and most probably later).

Nonetheless, thank you for responding on Christmas. :)
